docs/targets.rst

```diff
@@ -38,6 +38,8 @@ technology (LLVM) as the proprietary ARM compiler for code generation.
     <https://www.nordicsemi.com/eng/Products/Bluetooth-low-energy/nRF52832>`_)
   * `nRF52840-MDK <https://wiki.makerdiary.com/nrf52840-mdk/>`_ (`nRF52840
     <https://www.nordicsemi.com/eng/Products/nRF52840>`_)
+  * `Nordic PCA10056 <https://www.nordicsemi.com/eng/Products/nRF52840-DK>`_ (`nRF52840
```

Comment:
Weird, this page results in a 404 for me. But it exists in Google so I guess this is a temporary problem.

Reply:
Confirmed that is the correct board, and made change. Also corrected the link to the processor itself.